SQLScripts/InsertData_ATeam.sql

```diff
@@ -4,28 +4,34 @@ use Kainos_A;
 INSERT INTO band(bandName, bandExpectations)
 VALUES ("Apprentice", ""), ("Trainee", ""), ("Associate", ""),("Senior Associate", ""), ("Consultant", ""), ("Manager",""), ("Principle", ""), ("Leadership Community", "");
 
-insert into capability (capability) VALUES(("Engineering"),
-											("Platforms"));
+
+
+INSERT INTO capability (capability) VALUES(("Engineering"),("Platforms"));
```

Comment:
this thinks only trying to put in one ow cause of the outside ()s
```suggestion
INSERT INTO capability (capability) VALUES("Engineering"),("Platforms");
```